cobalt/configuration/configuration.cc

```diff
@@ -50,6 +50,9 @@ Configuration::Configuration() {
 
 const char Configuration::kSkiaRasterizer[] = "skia";
 const char Configuration::kGlesRasterizer[] = "direct-gles";
+const char Configuration::kEnableSkiaRasterizerPersistentSettingKey[] =
```

Comment:
Nice, getting those moved to a reasonable place. Maybe the constants should be more towards the top of the file as an easier to find place ? Around line 27 or so.